### PR TITLE
Labels spell build

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by 0xRobin** [comment]
I have mostly used labels in v1 to get reverse ENS resolution.
Here's a preliminary v2 query for that: https://dune.com/queries/1235389
(might need to add expiration or transfer checks though, not sure how those scenarios are handled within ENS, haven't done a deep dive in the contract)

Would be cool if we can get the 'ENS' and 'reverse ENS' labels into V2 :pray:

**Comment by soispoke** [comment]
Hey @0xRobin, yeah including ENS in labels would definitely be great. For now we're experimenting with labels on v2 but supporting ENS and reverse ENS is an important one. If you ever have time to deep dive and submit a PR for it that would be awesome and super useful for sure 🔥

**Review by soispoke — COMMENTED** [review]
@aalan3 @jeff-dude This PR is now ready for review

File: models/labels/funds/labels_funds_ethereum.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{config(alias='funds_ethereum')}}
+
+SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
+FROM (VALUES
+     (array('ethereum'),'0x2B1Ad6184a6B0fac06bD225ed37C2AbC04415fF4', 'a16z', 'funds', 'soispoke', 'static', timestamp('2022-10-03'), now())
```
Comment (jeff-dude) [inline models/labels/funds/labels_funds_ethereum.sql:5]: intended to be september 3rd?

File: models/labels/funds/labels_funds_schema.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    meta:
+      blockchain: ethereum
+      sector: labels
+      project: funds
+      contibutors: hildobby, soispoke
+    config:
+      tags: ['labels', 'ethereum', 'funds']
+    description: "Known funds addresses on Ethereum"
+    columns:
+      - name: blockchain
```
Comment (jeff-dude) [inline models/labels/funds/labels_funds_schema.yml:41]: should we use the '*' functionality to maintain consistency in the files?

File: models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{config(alias='gnosis_safe_ethereum')}}
```
Comment (jeff-dude) [inline models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql:1]: i believe the team has rebranded to just 'safe' now -- maybe we should reflect that in all folder/directories, file names, aliases, hardcoded columns, etc.

**Comment by jeff-dude** [comment]
most my feedback is just the hardcoded dates in future, renaming objects, using the '*' in schema file columns. otherwise, lgtm

File: models/labels/funds/labels_funds_ethereum.sql (hunk 1 above)
Comment (soispoke) [reply models/labels/funds/labels_funds_ethereum.sql:5]: Yes definitely 😅 I modified this in a commit

File: models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+SELECT
+    array('ethereum') as blockchain,
+    address,
+    'Gnosis Safe'  || ' version ' || creation_version AS name,
+    'safe' AS category,
+    'soispoke' AS contributor,
+    'query' AS source,
+    timestamp('2022-10-01') as created_at,
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql:10]: Yes same, this is now fixed

File: models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql (hunk 3 above)
Comment (soispoke) [reply models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql:1]: Oh good to know I wasn't sure, done ✔

File: models/labels/funds/labels_funds_schema.yml (hunk 2 above)
Comment (soispoke) [reply models/labels/funds/labels_funds_schema.yml:41]: Done, in all schema files ✔️

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**